src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html

```diff
@@ -121,7 +121,7 @@ <h3>
 Prior to 1.4, the helper threads were never terminated.
 <p>
 Starting with 1.4, the behavior has changed as a result of the fix for
-<a href="https://bugs.java.com/view_bug.do?bug_id=4030718">
+<a href="https://bugs.openjdk.org/browse/JDK-4030718">
```

Comment:
The old URL has changed to https://bugs.java.com/bugdatabase/view_bug.do?bug_id=4030718

I am for using JBS links though.